cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -95,6 +96,16 @@ var fRunAsConsole = flag.Bool("console", false, "run as console application (win
 var fSetEnv = flag.String("setenv", "", "set an env in the configuration file in the format of KEY=VALUE")
 var fStartUpErrorFile = flag.String("startup-error-file", "", "file to touch if agent can't start")
 
+// config-translator
+var fConfigTranslator = flag.Bool("config-translator", false, "run in config-translator mode")
```

Comment:
I spent some time trying to reduce the copy/paste aspect.  Some of these constants are defined twice.  There are some things to do to clean it up but they all end up less readable and add a lot of indirection.  Open to ideas tho.

If I was forced to make one change I would put the descriptions in a constant